### ISSUE TITLE
It's been more than a few weeks

### ISSUE THREAD

**Issue by hailwood** [markguinn/silverstripe-shop-groupedproducts#2]
Just sayin... ;) 

Seriously though, What state is this module in?

**Comment by adrian-stein** [comment]
I concur with ^^ . Is it in the pipeline at all to get it finished????

**Comment by markguinn** [comment]
Yeah, sorry guys. The project this was used in ended up being too site-specific to commit changes back directly and I haven't had time to do any more work on it.

That said, we're trying to pull together a core team to work on a 2.0 version of Shop and it's likely that one of the tasks will be expanding the product types to more closely match what's available in Magento, which would include taking this module or something like it across the line.

In the mean time, PR's are welcome and let me know if you'd like to be involved further with Shop 2.0.

**Comment by hailwood** [comment]
Hey @markguinn,

We're using the shop on a site roughly three weekly, so could be quite interested in assisting with Shop 2.0.

**Comment by markguinn** [comment]
@hailwood - great news. I've got a blog post coming out next week on SilverStripe.org outlining where we're at and how to participate. I'll keep you in the loop.
